### PR TITLE
Validate datasets

### PR DESCRIPTION
**Pull request by maukoquiroga**
partial_fix #26 
supersedes #107 

- [x] Tasks to seed, reset and validate data
- [x] Replaces JSON with MongoDB for dataset listing
- [x] Updates datasets with the corresponding data validation task id.

### CODE REVIEW

File: lib/mix/tasks/transport/seed.ex [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        description: "Horaires th\u00e9oriques du r\u00e9seau 'STAN' [PTU66 NANCY (P\u00e9rim\u00e8tre de transport urbain)]",
+        license: "fr-lo",
+        title: "Horaires th\u00e9oriques du r\u00e9seau de transport 'STAN'",
+        anomalies: [],
+        spatial: "Communaut\u00e9 Urbaine de Nancy",
+        logo: "https://www.data.gouv.fr/s/avatars/a3/4fddcd8a2441438a7d1ecb0dffa2dc-100.png",
+        slug: "horaires-theoriques-du-reseau-de-transport-stan",
+        format: "GTFS"
+      }
+    ]
```
Comment (l-vincent-l) [inline lib/mix/tasks/transport/seed.ex]: wouldn't it be more readable to just let this data in a json file and read it?

Comment (maukoquiroga) [reply lib/mix/tasks/transport/seed.ex]: For seeding the database? No...

Comment (maukoquiroga) [reply lib/mix/tasks/transport/seed.ex]: But we can do so if does make it more readable for you.

Comment (l-vincent-l) [reply lib/mix/tasks/transport/seed.ex]: It's just that a ex file doesn't seem a good place to store data to me.
I would prefer a json, or another file format that is used to store data.

File: lib/transport/reusable_data/reusable_data.ex [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  @pool DBConnection.Poolboy
+
+  @doc """
+  Returns the list of reusable datasets.
+
+  ## Examples
+
+      iex> ReusableData.list_datasets()
+      [%Dataset{title: "Leningrad metro dataset", anomalies: []}, ...]
```
Comment (l-vincent-l) [inline lib/transport/reusable_data/reusable_data.ex]: hehe

File: lib/mix/tasks/transport/seed.ex (hunk 1 above)
Comment (maukoquiroga) [reply lib/mix/tasks/transport/seed.ex]: Ok

**Pull request merged**